docs/servers/add-gcp-server.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Add a Server to GCP (Google Cloud Platform)
+intro: Devopness allows you to provision a server on Google Cloud Platform using
+Google Compute Engine and manage it through Devopness web application.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+# Add a Server to GCP (Google Cloud Platform)
+1. On Devopness, navigate to a project then select an environment
+2. If there's no other server in the selected environment, just click Add Server
+3. If a list of resource cards is displayed, find the Servers card and click View on it
+4. On the upper-right corner of the list click ADD SERVER
+5. Choose `Google Cloud Platform` as cloud provider and select its credentials
+6. Provide the instance type information and click NEXT
+7. Select your server name, OS and hostname and click NEXT
+8. Review server information and settings, click CONFIRM if everything is correct
```

Comment:
Other documentation files adopted the approach of highlighting UI element names, to catch user attention. E.g. `View`, `NEXT`, `CONFIRM`.
I believe all documents should apply a single pattern.
Please either use the existing docs as references for this one or modify existing ones to adhere to your new suggested pattern.
If you choose to apply new standards, please explain them in the PR description so the reviewers can understand the defense of your point of view.

Reply:
fixed
https://github.com/devopness/devopness/blob/2757da9e303846046d1b1ed042742ae8f401bd75/docs/servers/add-gcp-server.md?plain=1#L19-L23